src/core/agent/services/signifyNotificationService.ts

```diff
@@ -124,6 +127,26 @@ class SignifyNotificationService extends AgentService {
       a: result.content,
     };
   }
+
+  async dismissNotification(notificationId: string) {
+    const notificationRecord = await this.basicStorage.findById(notificationId);
+    if (!notificationRecord) {
+      throw new Error(SignifyNotificationService.KERI_NOTIFICATION_NOT_FOUND);
+    }
+    notificationRecord?.setTag("isDismissed", true);
```

Comment:
can we remove the `?` on this line - it's set because you would've thrown an error by now. I would just remove it in case we have a regression as it becomes harder to notice, and only use `?` where it's needed

Reply:
I've updated it